packages/react-table/src/demos/table-demos/StickyFirstColumn.jsx

```diff
@@ -0,0 +1,128 @@
+      detail3: `Test cell ${index + 1}-5`,
+      detail4: `Test cell ${index + 1}-6`,
+      detail5: `Test cell ${index + 1}-7`,
+      detail6: `Test cell ${index + 1}-8`,
+      detail7: `Test cell ${index + 1}-9`
+    })
+  );
+  const columnNames = {
+    name: 'Fact',
+    state: 'State',
+    header3: 'Header 3',
+    header4: 'Header 4',
+    header5: 'Header 5',
+    header6: 'Header 6',
+    header7: 'Header 7',
+    header8: 'Header 8',
+    header9: 'Header 9'
+  };
+  const [activeSortIndex, setActiveSortIndex] = React.useState(null);
+  const [activeSortDirection, setActiveSortDirection] = React.useState(null);
+  const getSortableRowValues = fact => {
+    const { name, state, detail1, detail2, detail3, detail4, detail5, detail6, detail7 } = fact;
+    return [name, state, detail1, detail2, detail3, detail4, detail5, detail6, detail7];
+  };
+  let sortedFacts = facts;
+  if (activeSortIndex !== null) {
+    sortedFacts = facts.sort((a, b) => {
+      const aValue = getSortableRowValues(a)[activeSortIndex];
+      const bValue = getSortableRowValues(b)[activeSortIndex];
+      if (aValue === bValue) {
+        return 0;
+      }
+      if (activeSortDirection === 'asc') {
+        return aValue > bValue ? 1 : -1;
+      } else {
+        return bValue > aValue ? 1 : -1;
+      }
+    });
+  }
+  const getSortParams = columnIndex => ({
+    sortBy: {
+      index: activeSortIndex,
+      direction: activeSortDirection
+    },
+    onSort: (_event, index, direction) => {
+      setActiveSortIndex(index);
+      setActiveSortDirection(direction);
+    },
+    columnIndex
+  });
+  return (
+    <DashboardWrapper hasPageTemplateTitle>
+      <PageSection isWidthLimited padding={{ default: 'noPadding', xl: 'padding' }}>
+        <Card component="div">
+          <InnerScrollContainer>
+            <TableComposable aria-label="Sticky column table" gridBreakPoint="">
+              <Thead>
+                <Tr>
+                  <Th isStickyColumn hasRightBorder modifier="truncate" sort={getSortParams(0)}>
+                    {columnNames.name}
+                  </Th>
+                  <Th modifier="truncate" sort={getSortParams(1)}>
+                    {columnNames.state}
+                  </Th>
+                  <Th modifier="truncate">{columnNames.header3}</Th>
+                  <Th modifier="truncate">{columnNames.header4}</Th>
+                  <Th modifier="truncate">{columnNames.header5}</Th>
+                  <Th modifier="truncate">{columnNames.header6}</Th>
+                  <Th modifier="truncate">{columnNames.header7}</Th>
+                  <Th modifier="truncate">{columnNames.header8}</Th>
+                  <Th modifier="truncate">{columnNames.header9}</Th>
+                </Tr>
+              </Thead>
+              <Tbody>
+                {sortedFacts.map(fact => (
+                  <Tr key={fact.name}>
+                    <Th isStickyColumn stickyMinWidth="100px" hasRightBorder modifier="truncate">
```

Comment:
This `MinWidth` needs to match the `thead > tr > th` `MinWidth`. Currently, one is `120px` and the other is `100px`. It doesn't really matter which value you choose as long as they're the same. Core uses `100px`.

Reply:
Nice catch @mattnolting , thank you! I've updated both `MinWidth`s to `100px` to match core.